Tasks/VsTest/distributedtest.ts

```diff
@@ -25,12 +25,12 @@ export class DistributedTest {
 
     public runDistributedTest() {
         this.publishCodeChangesIfRequired();
-        this.registerAndConfigureAgent();
+        this.beginDtaExecutionHost();
     }
 
     private publishCodeChangesIfRequired(): void {
         if (this.dtaTestConfig.tiaConfig.tiaEnabled) {
-            const code = testSelector.publishCodeChanges(this.dtaTestConfig.tiaConfig, null, this.dtaTestConfig.taskInstanceIdentifier);
+            const code = testSelector.publishCodeChanges(this.dtaTestConfig.tiaConfig, null, this.dtaTestConfig, null);
```

Comment:
why this change is required?

Reply:
proxy configuration is part part of both vstest config and dtaconfig. Since publish code changes is invoked in both the flows, we need to pass both.